jade/wsgi/routes/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from flask import render_template, send_file, request
+
+from . import v1
+
+PWD = os.path.dirname(os.path.abspath(__file__))
```

Comment:
`PWD` is a confusing name, since it usually refers to the shell's `cwd` at application launch time, which will not be the same directory we're talking about here.  Better to call this WSGI_CODE_DIR or something